lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -32,7 +32,14 @@
   name:
     description:
       - Name of the security group.
-    required: true
+      - One of and only one of I(name) or I(group_id) is required.
```

Comment:
I think it would be fine to specify that group_id can just be used with absent. The ability to change security group attributes with the ID could be another feature request/pr. If you're itching to implement that have at it, but do whatever makes sense to you.